doc/help/elektra-backends.md

```diff
@@ -26,7 +26,7 @@ It was impossible to implement powerful and feature-rich backends so far
 because of the lack of modularity. Desirable features like notification
 and type checking have always been in the developers' heads, but there was
 no place where it would fit in without making the system unmaintainable,
-complex and full of unwanted external dependences.
+complex and full of unwanted external dependencies.
```

Comment:
This is actually not an typo, both words exists. The question is: which on is the correct one?

Reply:
Since "dependencies" is the more common of the two in software development and this section describes software dependencies I would argue for the change.